examples/interop-examples/README.md

```diff
@@ -0,0 +1,102 @@
+# interop examples
+
+**WIP**
+
+## Run
+```
+cd examples/interop-examples
+npm i
+npm start
+```
+
+Open http://127.0.0.1:8080 in your favorite browser.
+
+## Tutorial
+
+Steps
+1. create IPFS instance
+
+First, we need an IPFS instance that we'll use throughout the tutorial. For the sake of convinience, we'll use `ipfs-daemon` module to create and IPFS instance.
```

Comment:
> I don't think we'll get to merge ipfs-daemon to js-ipfs within two weeks given how massive public API change that is. Let me know if you think we can.

We totally can, it is not actually that big of a change, really, just requires putting in the hours, it is pretty clear what we want out of it and we have a ton of tests creating js-ipfs daemons with different configurations and in Node.js + Browser, those that will ensure that the new API works.

It is, however, a low priority compared to the rest of the tasks for this Sprint, so, if we get to finish all the others, we will want ot tackle this.

Until then, the demo should be without an external module to create an instance of this module.